servicenow/plugin.spec.yaml

```diff
@@ -401,3 +401,27 @@ triggers:
         description: JSON object representing changed fields (map of field name to previous and current values)
         type: object
         required: true
+  incident_created:
+    title: Incident Created
+    description: Identifies if a new incident has been created
+    input:
+      query:
+        title: Query
+        description: Non-encoded query string to match new incident records (will poll for any new incident if query is omitted)
+        type: string
+        required: false
+        example: short_description=Newbug
+      interval:
```

Comment:
`frequency` is the variable name we use in triggers

https://komand.github.io/python/style.html#inputs